tests/tests/test_backward_compatibility/test_import.py

```diff
@@ -131,7 +164,11 @@ def test_23(verbose=False, plot=False):
         if parse(phoebe.__version__) >= parse("2.3.0") and parse(phoebe.__version__) < parse("2.4.0"):
             _export_23('23_export.phoebe')
             exit()
+        if parse(phoebe.__version__) >= parse("2.4.0") and parse(phoebe.__version__) < parse("2.5.0"):
+            _export_24('24_export.phoebe')
+            exit()
```

Comment:
do we need all exit()s here?

Reply:
no, but its convenient to just flip the one switch to call the script to create the bundles and skip the tests themselves 🤷. Alternatively I guess the tests could be in an else